src/components/NoteDialog/NoteDialog.test.tsx

```diff
@@ -128,5 +132,27 @@ describe("NoteDialog", () => {
       const {container} = render(createNoteDialog("Test Text", "Test Author", true), {container: global.document.querySelector("#portal")!});
       expect(container.querySelector(".note-dialog__note")?.children.item(1)).toHaveClass("note-dialog__footer");
     });
+
+    describe("Moderation phase", () => {
+      test("moderation: note-dialog__options note three isn't present", () => {
+        // @ts-ignore
+        Parse.User.current = jest.fn(() => ({id: "test-user-2"}));
+        const {container} = render(createNoteDialog("Test Text", "test-user-1", true, {userId: "test-user-1", status: true}), {
+          container: global.document.querySelector("#portal")!,
+        });
+        expect(container.querySelector(".note-dialog")?.children.item(3)?.children.length).toEqual(2);
+        expect(container.querySelector(".note-dialog")?.children.item(3)?.lastChild).not.toHaveClass("note-dialog__options");
```

Comment:
Same as https://github.com/inovex/scrumlr.io/pull/754/files#r723622834

Please write a more explicit query. If you struggle with the selection of a specific element you can also use the `data-testid` prop end select it via the testing library https://testing-library.com/docs/queries/bytestid/
But you should use that as your last option.